test/comparison/testdata/fbs/typeful/c/objectbox-model.h.expected

```diff
@@ -25,87 +25,96 @@ static inline OBX_model* create_obx_model() {
     
     obx_model_entity(model, "Typeful", 1, 8717895732742165505);
     obx_model_entity_flags(model, (OBXEntityFlags) (OBXEntityFlags_SHARED_GLOBAL_IDS | OBXEntityFlags_SYNC_ENABLED));
-    obx_model_property(model, "id", OBXPropertyType_Long, 1, 3390393562759376202);
+    obx_model_property(model, "id", OBXPropertyType_Long, 1, 2669985732393126063);
```

Comment:
I've tried moving to another .fbs but it doesn't help - still ends up with new IDs because these files are always regenerated. The only option is adding another "directory" to the comparison test (i.e. it wouldn't be in `typeful` but `enums`) but you didn't want that previously. 

BTW the first commit is just adding the enum to the model. the second commit is the actual changes - maybe ignoring the first commit helps reviewing